### ISSUE TITLE
missing scripts

### ISSUE THREAD

**Issue by splaisan** [BrendelGroup/iLoci_SLB22NARGB#9]
Hi,

I want to compare my local assembly/annotation to that of the reference genome. For that you pointed me to this repo and I am trying to adapt the tutorial to my data.

At the bottom of https://github.com/BrendelGroup/iLoci_SLB22NARGB/tree/main/work/comparisons, the link to the python visualisation script is pointing to https://github.com/timlai4/IntervalLoci/blob/comparisons/compare/comparison_plots.ipynb

Later in the comparison (iLoci_SLB22NARGB/work/comparisons/README.md) another script 'parse_loci.py' is used but not found on the repo

Running the lastz commands adapted from https://github.com/BrendelGroup/iLoci_SLB22NARGB/blob/main/work/comparisons/runAmel.sh leads to errors

```
workdir="/data/analyses/genome_assembly/euk/Chlamydomonas_reinhardtii/annotation_results/aegean_results/compare"
cd ${workdir}
# get the code from github
git clone https://github.com/BrendelGroup/iLoci_SLB22NARGB.git
cp -r iLoci_SLB22NARGB/work/comparisons/* .
# make alias for singularity
alias rws='singularity exec -e -B ${workdir}:/workdir /opt/sif_files/aegean.sif '

# run scripts/split_iloci_by_type.py on both genomes 

# try one lastz command
rws lastz /workdir/Gnm1.iloci.fa /workdir/Crei.iiloci.fa --allocate:traceback=500M --ambiguous=iupac --filter=identity:95 --chain --format=general:name1,length1,size1,name2,length2,size2,identity,nmatch > Gnm1_vs_Crei_iiloci.lastz
FAILURE: target file /workdir/Gnm1.iiloci.fa contains more than one sequence
consider using the "multiple" action (see "lastz --help=files")
```

Thanks for your help in completing the necessary tools to reproduce your work and use it on other data

Best
S

**Comment by vpbrendel** [comment]
Hi Stephane:
You are right, the documentation is outdated. Correct are the runA*sh scripts (as you found).
To fix your lastz error, you need to add [multiple] after the query, as in our script, ie.

rws lastz /workdir/Gnm1.iloci.fa[multiple] ...

Volker

**Comment by splaisan** [comment]
Dear Volker,

Thanks you so much for your patience and help.
I fixed the command as you pointed out and indeed it worked with the code from the shell script.
I found that most of my annotations fall under unmapped when compared to the reference genome, probably because the new assembly annotations differ from the original (one to many lastz relationships) of my assembly has a lot of indels (ONT).

I will try to figure out what to do with this but at least now it ran from your code.

I guess I only need to consider the last lines of the 5 summary files and create barplots or tables from that data. 

Best regards,
Stephane
